agent/runtime-attach/src/main/java/com/microsoft/applicationinsights/attach/ApplicationInsights.java

```diff
@@ -17,6 +17,9 @@
 /** This class allows you to attach the Application Insights agent for Java at runtime. */
 public final class ApplicationInsights {
 
+  public static final String APPLICATIONINSIGHTS_RUNTIME_ATTACH_CONFIGURATION_FILE =
+      "applicationinsights.runtime-attach.configuration.file";
```

Comment:
can we make this work for a file path as well?

Reply:
are we going with "applicationinsights.runtime-attach.configuration.classpath.file"? can we add a comment to say this file must be in the classpath, otherwise it wouldn't work?